python/src/opendp/extras/polars/__init__.py

```diff
@@ -611,16 +612,22 @@ def _lazyframe_domain_from_schema(schema) -> Domain:
 
 def _series_domain_from_field(field) -> Domain:
     """Builds the broadest possible SeriesDomain that matches a given field."""
+    name, dtype = field
+    return series_domain(name, option_domain(_domain_from_dtype(dtype)))
```

Comment:
Follow-up to the comment on the test: This is where the `option_domain` comes from, but I don't really understand why it's needed.

Reply:
Yep, input data could have null rows (or at least, a potential adjacent dataset could have a new record with null in the array column).